File: P10_1403/P10_1403.c
#include <stdio.h>

int main()
{
    int n;
    printf("Enter length of array: ");
    scanf("%d", &n);

    int arr[n];

    int i = 0;
    for (i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }

    int zeros = 0;
    int ones = 0;

    for (i = 0; i < n; i++)
    {
        if (arr[i] == 0)
            zeros++;
        else
            ones++;
    }

    i = 0;
    while (zeros)
    {
        arr[i] = 0;
        zeros--;
        i++;
    }

    while (ones)
    {
        arr[i] = 1;
        ones--;
        i++;
    }

    printf("\nOutput:\n");
    for (i = 0; i < n; i++)
    {
        printf("%d ", arr[i]);
    }
}

File: P10_1403/P10_1403.py
n = int(input())
l = []

for i in range(n):
    l.append(int(input()))

ones = 0
zeros = 0

for i in l:
    if i == 0:
        zeros += 1
    else:
        ones += 1
i = 0

while(zeros):
    l[i] = 0
    i += 1
    zeros -= 1

while(ones):
    l[i] = 1
    i += 1
    ones -= 1

for i in l:
    print(i, end=" ")
